src/pages/App.tsx

```diff
@@ -86,8 +87,10 @@ export default function App() {
             <Polling />
             <TopLevelModals />
             <Switch>
-              <Route exact strict path="/vote" component={Vote} />
-              <Route exact strict path="/vote/:governorIndex/:id" component={VotePage} />
+              <Suspense fallback={<Loader />}>
+                <Route exact strict path="/vote" component={Vote} />
+              </Suspense>
```

Comment:
Considering that other routes may be lazy-loaded in the future, I think it's appropriate to wrap the Switch in Suspense, instead of just the route. wdyt?

Reply:
Sure, I just assumed you might want to specify different loading indicators for different sections.